.drone.star

```diff
@@ -2412,6 +2456,20 @@ def rebuildBuildArtifactCache(ctx, name, path):
 def purgeBuildArtifactCache(ctx):
     return genericBuildArtifactCache(ctx, "", "purge", [])
 
+def restoreGoCache(ctx, name, path):
+    return restoreBuildArtifactCache(ctx, name, path) + [
+        {
+            "name": "move-go-cache",
+            "image": OC_CI_PHP % DEFAULT_PHP_VERSION,
+            "commands": [
+                "rsync -a --remove-source-files %s/go/ /go" % dirs["base"],
+                "ls -al -r %s/go" % dirs["base"],
+                "rm -rf %s/go" % dirs["base"],
+            ],
+            "volumes": [stepVolumeGo],
+        },
+    ]
```

Comment:
the cache restore path is relative to the current drone workdir. so the cache is restored to `/drone/src/go` but we need it to the root path (`/go`). One way is to restore and move the cache to the desired location like I did here.
 
Other options could be (but could be troublesome):
- set GOPATH to `/drone/src/go`
- change the workdir to '/'

CC @individual-it

Reply:
the move seems to be reasonable, but why using rsync and not a simple `mv`?

Reply:
Tried different mv command but could not get it working:
1. 
```bash
+ mv /drone/src/go /
mv: inter-device move failed: '/drone/src/go' to '/go'; unable to remove target: Device or resource busy
```
2.
```bash
+ mv /drone/src/go/{.,}* /go/
mv: cannot stat '/drone/src/go/{.,}*': No such file or directory
```
Those worked on my machine but not in drone.

Reply:
probably you cannot move between volumes